src/pages/webinar/[webinarSlug]/index.tsx

```diff
@@ -13,6 +13,8 @@ import { useApi, useUser } from '@/hooks';
 import { routes } from '@/constant';
 import { WebinarHeroContainer } from '@/components';
 import { isProgramActive } from '@/utils';
+import { imageMeta } from '@/constant/global';
+import LinkButton from '@/components/common/Buttons/LinkButton';
```

Comment:
always use Root address like @/constant or @/components not @/components/common/Buttons/LinkButton